dotnet/src/Skills/Skills.Web/Bing/BingConnector.cs

```diff
@@ -108,15 +133,15 @@ private sealed class BingSearchResponse
 
     [SuppressMessage("Performance", "CA1812:Internal class that is apparently never instantiated",
         Justification = "Class is instantiated through deserialization.")]
-    private sealed class WebPages
+    public sealed class WebPages
```

Comment:
Can you clarify why these should be private please?

I've opted to expose these classes, to enable the caller to be able to use the class objects in calls & returned output.

Reply:
We should be careful what we expose to SDK users. If we make some class `public`, it means that SDK user will get access to it. In this case we should think about more specific naming (e.g. `BingWebPage`), add detailed XML documentation what this class is responsible for and provide documentation for each public member. And most importantly - we need to think whether we should provide this class to the end users at all. 

As I mentioned in previous comment, Kernel doesn't know how to work with this class (yet). Which means that it will be useful only when you call this skill directly without a Kernel. In this case it will look like we are developing Bing SDK, which is not the case. This skill should be useful in a context of Semantic Kernel, rather than separately.